pkg/controller/grouping/custom_label.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package grouping
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+const (
+	CustomLabelKeyPrefix   = "antrea.io/reserved-label-"
```

Comment:
I wonder if we really need a "reserved-label" prefix given that no one else should be defining `antrea.io` labels. What do you think of `internal.antrea.io/service-account`?

Reply:
Sounds good.